packages/react-components/src/DocsComponents/Toc.stories.tsx

```diff
@@ -59,7 +58,7 @@ const useTocStyles = makeStyles({
   },
 });
 
-type TocItem = PublishedStoreItem & { selected?: boolean };
+type TocItem = { name: string; id: string; selected?: boolean };
```

Comment:
This typing no longer matched, and I couldn't find the componentStories return type being exported. name and ID are all we use in the TOC anyway.